apps/s_client.c

```diff
@@ -3553,4 +3556,70 @@ static char *base64encode (const void *buf, size_t len)
     return out;
 }
 
+/*
+ * Host dNS Name verifier: used for checking that the hostname is in dNS format 
+ * before setting it as SNI
+ */
+static int is_dNS_name(const char *host)
```

Comment:
Cosmetic: (@mattcaswell do we care?) We don't normally camel case internal functions in OpenSSL.  To comport with the style of the existing code, perhaps this should be just `is_dns_name()`?  Secondly, perhaps this function should be in `apps.c` just in case some other app will find use for it?  I'm approving the PR, but if Matt or someone else wants the style issued addressed, please speak up...

Reply:
There are other names like this already in this file (e.g. `ldap_ExtendedResponse_parse`) so I don't find it so jarring in this case. I think the function is fine in this file for now. We can always move it to apps.c later if appropriate.